### ISSUE TITLE
Raise error when plotting for variable that is unknown to an observables

### ISSUE THREAD

**Issue by dvandyk** [eos/eos#345]
@cBobeth reports that plotting an observables as a function of a kinematic variable that is not known to the observable presently does not raise an error.

**Comment by dvandyk** [comment]
After discussions @cBobeth proposes to retire the keys ``parameter`` and ``kinematic`` and only keep ``variable``, which can be either the qualified name for an ``eos.Parameter`` or the name of an ``eos.KinematicVariable`` known to the observable. The latter is discoverable through the observable's ``eos.ObservableEntry``.

Example
``` python
# test if a variable 'vname' is known to an observable 'oname'
oentries = eos.Observables
(vname in oentries[oname].kinematic_variables()) if oname in oentries
```

**Comment by mreboud** [comment]
Should be solved with #346.

**Comment by pluegh** [comment]
#346 is merged into master.
